### PR TITLE
New Docs

### PR DESCRIPTION
**Pull request by quarties**

### CODE REVIEW

File: docs/Introduction.md [hunk 1]
```diff
@@ -1,15 +1,25 @@
...
+We created this **Design System** with several goals in mind:
 
-Private repository is available on github: [livechat/design-system](https://github.com/livechat/design-system).
+- to document components, patterns, and design guidelines, all in one place,
+- to ensure consistency in our code and design,
+- to standardize the visual language and experience of LiveChat products and apps,
+- to provide guidance on correct usage of the patterns,
+- to streamline our design and development.
 
-Static css file is hosted on CDN:
```
Comment (kamilmateusiak) [inline docs/Introduction.md]: we should include this information in our docs

File: docs/customized-components/MenuList.js [hunk 2]
```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+/* eslint-disable import/no-unresolved */
```
Comment (kamilmateusiak) [inline docs/customized-components/MenuList.js]: interesting :D please run `npm run build` and check if this works after build

File: docs/customized-components/Button.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Button as DSButton } from '../../src/index';
+
+function Button({ name, className, onClick, active, children }) {
+  return (
+    <DSButton
+      className={className}
```
Comment (kamilmateusiak) [inline docs/customized-components/Button.js]: how about `compact` buttons here?

File: src/docs/ComponentHtmlMarkup.js [hunk 4]
```diff
@@ -43,19 +43,13 @@ class ComponentHtmlMarkup extends React.Component {
...
-          className={cx({
-            'view-html-btn': true,
-            'view-html-btn--active': this.state.isCodeVisible
-          })}
-          onClick={this.toggleCodeVisibility}
-        >
-          View html code
-        </button>
-        {this.state.isCodeVisible && (
+        <Button onClick={this.toggleCodeVisibility} primary={isCodeVisible}>
```
Comment (kamilmateusiak) [inline src/docs/ComponentHtmlMarkup.js]: `compact` - this is too large imho

File: docs/customized-components/Menu.js [hunk 5]
```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Styled from 'rsg-components/Styled'; // eslint-disable-line import/no-unresolved
```
Comment (dprzybylek) [inline docs/customized-components/Menu.js]: why this ` eslint-disable-line` is necessary here?

File: docs/customized-components/MenuList.js (hunk 2 above)
Comment (quarties) [reply docs/customized-components/MenuList.js]: tested locally and works fine

File: docs/customized-components/Menu.js (hunk 5 above)
Comment (quarties) [reply docs/customized-components/Menu.js]: those components are available however, those are not part of package.json - that's why eslint is angry

**Pull request merged**